examples/main/main.cpp

```diff
@@ -113,6 +113,17 @@ int main(int argc, char ** argv) {
         }
     }
 
+    if (!params.lora_adapter.empty()) {
+        int err = llama_apply_lora_from_file(ctx,
```

Comment:
Can we add a check here if the -m model is q4 and also recommend to pass `--lora-base` as the workaround?

Reply:
Was just working on that, it is done already.